backend/kernelCI_app/models.py

```diff
@@ -228,3 +228,73 @@ class Meta:
             models.Index(fields=["issue_version"], name="incidents_issue_version"),
             models.Index(fields=["origin"], name="incidents_origin"),
         ]
+
+
+class HardwareStatus(models.Model):
+    checkout_id = models.TextField()
+    origin = models.CharField(max_length=100)
+    platform = models.CharField(max_length=100)
+    compatibles = ArrayField(models.TextField(), null=True)
+    start_time = models.IntegerField()
+
+    # build status
+    build_pass = models.IntegerField(default=0)
+    build_failed = models.IntegerField(default=0)
+    build_inc = models.IntegerField(default=0)
+
+    # boot status
+    boot_pass = models.IntegerField(default=0)
+    boot_failed = models.IntegerField(default=0)
+    boot_inc = models.IntegerField(default=0)
+
+    # test status
+    test_pass = models.IntegerField(default=0)
+    test_failed = models.IntegerField(default=0)
+    test_inc = models.IntegerField(default=0)
+
+    class Meta:
+        db_table = "hardware_status"
+        unique_together = ("checkout_id", "origin", "platform")
```

Comment:
I'd have origin, platform first so indexes reaching only them will work.

also need another index on origin, platform, start_time... since we also query based on those.